lib/transformers/govukDateInput.js

```diff
@@ -38,6 +40,7 @@ module.exports = ({schemaKey, schema, options, data, schemaErrors} = {}) => {
                 {
                     label: 'Year',
                     classes: `govuk-input--width-4`,
+                    id: `${schemaKey}[year]`,
```

Comment:
This is to address [this](https://github.com/CriminalInjuriesCompensationAuthority/q-transformer/blob/master/lib/transformers/govukDateInput.js#L72) issue